babi/__init__.py

```diff
@@ -0,0 +1,13 @@
+def _version() -> str:
+    try:
+        from importlib import metadata  # type: ignore
+    except ImportError:
+        import importlib_metadata as metadata  # type: ignore
+
+    return metadata.version('babi')
```

Comment:
this is how I would do this:

https://github.com/pre-commit/pre-commit/blob/528c7afd18dafa6e47ce73add2c8e1550d105674/pre_commit/constants.py#L3-L6